Gemfile

```diff
@@ -75,3 +75,9 @@ gem 'will_paginate'
 gem 'devise', '~> 4.9'
```

Comment:
![image](https://github.com/stino-x/Blog-Rails-/assets/93039359/b420864b-79b5-4085-897a-733759268452)

- [ ] It would be nice if you fix the highlighted link in the image above :eyes: because right now it is broken :fearful: Kindly remove it if you don't have any dedicated section or fix it to make your readme descriptive and professional :+1: .